### PR TITLE
Example section update

### LINKED ISSUE #2692: "very similar" seems "exactly the same" and only mentioned at 3.3.8
https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-no-exception.html#examples

3.3.8 Examples starts with a sentence:

> The examples of this success criterion are very similar to the [Accessible Authentication](https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication.html#examples).

Maybe I don't see the difference but I think they are "exactly the same" not "very similar"

Also 3.3.8 has the sentence while 3.3.7 does not, should be consistently referring to each other, adding the same at 3.3.7

https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication.html#examples

### PR DESCRIPTION
**Pull request by cwadamsoforacle**
Closes #2692

### CODE REVIEW

File: understanding/22/accessible-authentication.html [hunk 1]
```diff
@@ -138,6 +138,7 @@ <h2>Benefits of Accessible Authentication</h2>
    
    <section id="examples">
       <h2>Examples of Accessible Authentication</h2>
+      <p>The examples of this success criterion are the same as the <a href="accessible-authentication-enhanced.html#examples">Accessible Authentication (Enhanced)</a> examples.</p>
```
Comment (mbgower) [inline understanding/22/accessible-authentication.html:141]:
```suggestion
      <p>The examples of this Success Criterion are the same as the <a href="accessible-authentication-enhanced.html#examples">Accessible Authentication (Enhanced)</a> examples.</p>
```

File: understanding/22/accessible-authentication-enhanced.html [hunk 2]
```diff
@@ -68,7 +68,7 @@ <h2>Benefits of Accessible Authentication (Enhanced)</h2>
    
    <section id="examples">
       <h2>Examples of Accessible Authentication (Enhanced)</h2>
-      <p>The examples of this success criterion are very similar to the <a href="accessible-authentication.html#examples">Accessible Authentication</a>.</p>
+      <p>The examples of this success criterion are the same as the <a href="accessible-authentication.html#examples">Accessible Authentication</a> examples.</p>
```
Comment (mbgower) [inline understanding/22/accessible-authentication-enhanced.html:71]:
```suggestion
      <p>The examples of this Success Criterion are the same as the <a href="accessible-authentication.html#examples">Accessible Authentication</a> examples.</p>
```

**Comment by rachaelbradley** [comment]
Accepted at 4/11/23 AG meeting.

**Review by rachaelbradley — APPROVED** [review]
Approved at AG meeting on 11 April 2023

**Pull request merged**